### PR TITLE
Bucharest move post

### PR DESCRIPTION
**Pull request by SathyaBhat**

### CODE REVIEW

File: content/blog/2019-11-17-salut-bucharest.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+So some of the obvious questions would be:
+
+- Why Romainia?
+- How did it happen?
+
+Let's take on them one by one:
+
+### Why Romania?
+
+To which I say why not? I've been looking at moving out for a while now, and the US was never an option, especially with the current visa situation(and Jo not being able to work with a dependent visa). Europe is what Jo and me had been looking at since a while. I've been working with my colleagues from Bucharest, Romania since past couple of couple of years and this gave me a chance to bring up a internal transfer request within my team. With the approvals in place, we were finally able to start the process. However, as with immigration & inter-country transfers, this was just the start of a long road of lots of paperwork and documentation shuttle.
```
Comment (ninadsp) [inline content/blog/2019-11-17-salut-bucharest.md]: You might want to add a space between `situation` and, `(and Jo`. Also, the next sentence has `couple of ` twice.

File: content/blog/2019-11-17-salut-bucharest.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+### How did it happen?
+
+Immigration and work visas are never easy, and Romania's visa/work permit process wasn't a piece of cake either. Before the visa process could begin, I had to get a work permit - which can be only done by the employer. And for applying the work permits, I had to get a "Criminal Record" aka PCC/Police Clearance Certificate, Transcripts from my University, my Degree certificate and my previous experience certificates - and I had to get them [Apostilled](https://en.wikipedia.org/wiki/Apostille_Convention) too!
+
+One of the many TIL moments was about Apostille.. 
+
+> A certification under the terms of the Apostille Convention is called an apostille (from Latin post illa and then French: a marginal note) or Hague Apostille. It is an international certification comparable to a notarisation in domestic law, and normally supplements a local notarisation of the documen
```
Comment (ninadsp) [inline content/blog/2019-11-17-salut-bucharest.md]: Missed the last `t` for document :)

File: content/blog/2019-11-17-salut-bucharest.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+The good thing was most of these were well calatalogued and saved(I have almost everything digitised with multiple backups) and the ones that needed to be done, namely the transcripts and PCC, I could apply for these online.
```
Comment (ninadsp) [inline content/blog/2019-11-17-salut-bucharest.md]: `catalogued`? There's a bit here where there could be a space before the bracket again...

File: content/blog/2019-11-17-salut-bucharest.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+However, this bit of information is still not well known and the embassy website still encourages first-come, first-serve - and if you follow this, like me, you'll just end up wasting a whole lot of time. Another pain point is that even though the embassy now encourages visiting only after getting an appointment, there doesn't seem to be any automated way of getting an appointment - when I visited the embassy and stood for hours, the embassy official noted my file number and scheduled for an appointment for 5 days later. I wonder if/when I would have gotten the appointment, had I not gone and stood at the embassy - especially since the work permits are time bound - ie, the visa application for the work permit must be done within 2 months of the work permit being approved else it excpires and a new work permit must be applied for - which needs a new PCC because that also has a limited eligiblity window - you see where this is going?
```
Comment (ninadsp) [inline content/blog/2019-11-17-salut-bucharest.md]: Expires has an extra `c` in there...

File: content/blog/2019-11-17-salut-bucharest.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+<!--more-->
+
+
+{{< tweet 1182251512334077952 >}}
+
+{{< tweet 1184008163546284032 >}}
+
+So some of the obvious questions would be:
+
+- Why Romainia?
```
Comment (ninadsp) [inline content/blog/2019-11-17-salut-bucharest.md]: Is this a pun about Romania being the main thing in your life for the past few months? :P

File: content/blog/2019-11-17-salut-bucharest.md (hunk 3 above)
Comment (SathyaBhat) [reply content/blog/2019-11-17-salut-bucharest.md]: lol at "calatalogued". Fixed

File: content/blog/2019-11-17-salut-bucharest.md (hunk 4 above)
Comment (SathyaBhat) [reply content/blog/2019-11-17-salut-bucharest.md]: fixed

File: content/blog/2019-11-17-salut-bucharest.md (hunk 5 above)
Comment (SathyaBhat) [reply content/blog/2019-11-17-salut-bucharest.md]: haha maybe I should keep it as here :P

**Pull request merged**